custom/icds_reports/data_pull/consolidate_monthly_report.py

```diff
@@ -0,0 +1,48 @@
+import csv
+from copy import copy
+from collections import defaultdict
+
+STATE_NAME_COLUMN = "state_name"
+filenames = [
+    'monthly_stats1.csv',
+    'monthly_stats2.csv',
+    'monthly_stats3.csv',
+    'monthly_stats4.csv'
+]
+
+test_state_names = [
+    'Demo State',
+    'Uttar Pradesh_GZB',
+    'VL State',
+    'Trial State',
+    'Test State 2',
+    'AWW  State Testing',
+    'Practice Mode State',
+    'Test State'
```

Comment:
Hey, new test states can be added in the future and names can change. So it will be good to filter them out on DB level itself. This could be usefull here : https://github.com/dimagi/commcare-hq/pull/26111#discussion_r353545129

Reply:
Is it true that `state_is_test` column is available for some tables and not for all.
If thats the case, we would still need to keep the filtering in python but filtering here would help reduce the result data. So will add that